routes/scenario_routes.py

```
@@ -0,0 +1,522 @@
+# セキュリティ関連のインポート
+try:
+    from utils.security import SecurityUtils
+except ImportError:
+
+    class SecurityUtils:
+        @staticmethod
+        def sanitize_input(text):
+            return text
+
+        @staticmethod
+        def validate_scenario_id(scenario_id):
```

Ruff (0.14.6)

Unused static method argument: `scenario_id`

(ARG004)